zerver/openapi/markdown_extension.py

```diff
@@ -127,6 +127,13 @@ def curl_method_arguments(endpoint: str, method: str,
 
 def get_openapi_param_example_value_as_string(endpoint: str, method: str, param: Dict[str, Any],
                                               curl_argument: bool=False) -> str:
+    jsonify = False
+    param_name = param["name"]
+    if "content" in param:
+        param = param["content"]
+        if "application/json" in param:
+            jsonify = True
+        param = param[list(param.keys())[0]]
```

Comment:
Similarly here, we should not pretend to understand what’s going on if the type is anything other than `application/json`, we should just crash immediately.

Reply:
Added assert statement.